gradle/verification-metadata.xml

```diff
@@ -35,6 +35,31 @@
             <pgp value="7615ad56144df2376f49d98b1669c4bb543e0445"/>
          </artifact>
       </component>
+      <component group="com.google.guava" name="failureaccess" version="1.0.1">
```

Comment:
Those entries shouldn't be necessary and are covered by the accepted keys above. Is that not the case? I think adding only the conventions plugin should be enough.

Reply:
OK. I wonder why the automated task doesn't just skip them then...